apps/joinmarket-webui/docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: "3.7"
+
+services:
+  jam:
+    image: ghcr.io/joinmarket-webui/joinmarket-webui-standalone:v0.0.3-clientserver-v0.9.5@sha256:d33817e4daec4ddaaf95a1c08e54f50cff4893b7d8c5b3bbe8b179ced74bbaa2
```

Comment:
Did you miss the name change in the Docker image or is it not renamed to `jam-standalone` (or something like that)?

Reply:
No, image is already pinned. Everything will be renamed (repos, support group, hyperlinks in general, etc.) within the next releases. Since it is not user-facing - is it okay to leave it that way?

Reply:
Yeah that's fine.